contributors/devel/testing.md

```diff
@@ -54,6 +54,12 @@ cd kubernetes
 make test  # Run all unit tests.
 ```
 
+if any unit test fails with a timeout panic (see #1594) on the testing package. you can increase the KUBE_TIMEOUT value as shown below.
```

Comment:
"If" and "You" should be capitalized at the beginning of sentences.

Reply:
Can you add a link to the issue number?
Something like this:
```
(see [#1594](https://github.com/kubernetes/community/issues/1594))
```

Reply:
Please wrap `KUBE_TIMEOUT` in single backticks.

Reply:
Thank you for the review, I have made the changes.